pkg/securityhubcollector/security_hub_collector.go

```diff
@@ -178,7 +193,7 @@ func (h *HubCollector) writeHeadersToOutput() error {
 	// out the data we wanted from these findings changed regularly, we
 	// could make the headers/fields come from some sort of schema or struct,
 	// but for now this is good enough.
-	headers := []string{"Team", "Resource Type", "Title", "Description", "Severity Label", "Remediation Text", "Remediation URL", "Resource ID", "AWS Account ID", "Compliance Status", "Record State", "Workflow Status", "Created At", "Updated At"}
+	headers := []string{"Team", "Resource Type", "Title", "Description", "Severity Label", "Remediation Text", "Remediation URL", "Resource ID", "AWS Account ID", "Region", "Environment", "Compliance Status", "Record State", "Workflow Status", "Created At", "Updated At"}
```

Comment:
did you consider adding these fields to the end so that tools that are expecting the old format would still work?